### PR TITLE
[4.1.0] Add heap dump docs for CC Router

### PR DESCRIPTION
**Pull request by renuka-fernando**
## Purpose
$subject
Related to https://github.com/wso2/product-microgateway/issues/2997

Add following fields to mkdocs.yaml file, since there are many places Choreo Connect version is used in docs (for docker images)

```yaml
    choreo_connect:
        version: 1.1.0
        envoy_version: v1.20.2
        helm_chart:
            version: 1.1.0-3
            git_tag: 1.1.0.3
```

image

### CODE REVIEW

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/debug-memory.md [hunk 1]
```diff
@@ -37,3 +37,113 @@ Follow the steps below to take a heap dump of the Adapter.
 ## Taking a heap dump of Enforcer
 
 Enforcer is implemented in Java and it is configured with required parameters to dump the heap information when out of memory errors happen. The dump can be collected from `/home/wso2/logs/heap-dump.hprof`
+
+## Taking a heap dump of Router
+
+Router uses the Envoy Proxy as the core component that does the traffic routing. The default Router which is used in you production environment is not supporting heap profiling and we have build a separate Router Docker image which enables heap profiling. Follow the steps below to take a heap profile of the Router.
```
Comment (pubudu538) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/debug-memory.md:43]: We have built

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/debug-memory.md [hunk 2]
```diff
@@ -37,3 +37,113 @@ Follow the steps below to take a heap dump of the Adapter.
...
+
+### Prerequisites
+
+1.  Install [pprof](https://github.com/google/pprof#readme) and [Graphviz](http://www.graphviz.org/).
```
Comment (pubudu538) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/debug-memory.md:47]: We should specify where to install and why we need to install it. eg: To analyze the heap dump you need to have pprof.. and you can install in your local machine...

Comment (renuka-fernando) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/debug-memory.md]: fixed with https://github.com/wso2/docs-apim/pull/6067/commits/83e49a43e873b2b537d601f13df397e4b6e72c31

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/debug-memory.md (hunk 1 above)
Comment (renuka-fernando) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/debug-memory.md]: fixed with https://github.com/wso2/docs-apim/pull/6067/commits/83e49a43e873b2b537d601f13df397e4b6e72c31

**Review by pubudu538 — APPROVED** [review]

**Pull request merged**